training/statistics/machinelearning/machine_learning-test.yml

```diff
@@ -0,0 +1,16 @@
+---
+- doc: Test sample data for Intro to Machine Learning workflow
```

Comment:
The doc string is actually for the entire test not only the test data.

Reply:
Good point.  I've updated it to 'Test Basics of Machine Learning workflow'